packages/vite/src/node/config.ts

```diff
@@ -440,6 +440,12 @@ export interface UserConfig extends DefaultEnvironmentOptions {
    * @default 'spa'
    */
   appType?: AppType
+  /**
+   * Specifies the number of times to recurse while formatting object.
+   *
+   * https://nodejs.org/api/util.html#utilinspectobject-options
+   */
+  debugDepth?: string | undefined
```

Comment:
The environment variable stores a string, and the `util.inspect` option accepts a number. I am not sure whether the type here should be set to a number or a string. Currently, it is set to a string.